### PR TITLE
EMAILPLT-192: Add README and add link to uPortal contibuting guide

### PR DESCRIPTION
**Pull request by ChristianMurphy**
Ticket: https://issues.jasig.org/browse/EMAILPLT-192
Ported from: https://wiki.jasig.org/display/PLT/Email+Preview+Portlet

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Requested in https://github.com/Jasig/email-preview/pull/71#pullrequestreview-20571763

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Guidelines for Contributing
+
+This project follows the [Apereo Contributing Guidelines](https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md).
```
Comment (apetro) [inline CONTRIBUTING.md]: This link over-states the scope of `Jasig/uPortal/CONTRIBUTING.md`. At most `Jasig/uPortal/CONTRIBUTING.md` is uPortal-ecosystem contributing guidelines.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+*   Mailbox folder name
+*   Mark Messages Read
+*   Username (portletPreferences authentication only)
+*   Password (portletPreferences authentication only)
+*   Show rollup (smaller) or preview (larger) on login
+*   Focus on Preview
+
+## Contributing
+
+This project follows the [Apereo Contributing Guidelines](https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md).
```
Comment (apetro) [inline README.md]: Ditto on the "Apereo" rather than "uPortal" being too broad a scope grab.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ChristianMurphy) [inline CONTRIBUTING.md]: Updated

File: README.md (hunk 2 above)
Comment (ChristianMurphy) [inline README.md]: Updated

**Pull request merged**